.github/workflows/promote_charm.yaml

```diff
@@ -8,11 +8,13 @@ on:
         description: 'Origin Channel'
         options:
         - latest/edge
+        - 1/edge
       destination-channel:
         type: choice
         description: 'Destination Channel'
         options:
         - latest/stable
+        - 1/stable
```

Comment:
Just a note, this can potentially allow us to promote from latest/edge to 1/stable

Reply:
Good point, changed to allow only 1/edge to 1/stable.